### ISSUE TITLE
How are the asymmetric value clip bounds `[-5, 10]` determined in PPO?

### ISSUE THREAD

**Issue by falenai** [zhaozijie2022/LocoWheeledLegged#7]
In `loco_rl/algorithms/ppo.py` (lines 307-315), the value function outputs, returns, and target values are all clamped to an asymmetric range:

```python
clip_value_min: float | None = -5.0
clip_value_max: float | None = 10.0
```

```python
if self.clip_value_min is not None and self.clip_value_max is not None:
    returns_batch = torch.clamp(returns_batch, self.clip_value_min, self.clip_value_max)
    target_values_batch = torch.clamp(target_values_batch, self.clip_value_min, self.clip_value_max)
    value_batch = torch.clamp(value_batch, self.clip_value_min, self.clip_value_max)
```

A few questions:

1. **Why clamp the absolute value at all?** Standard PPO uses relative clipping on the value update (`value_clipped = target + clamp(value - target, -eps, eps)`), which is already implemented below this block. The absolute clamp seems like an additional safety mechanism -- is it to prevent value explosion from rare outlier returns?

2. **Why asymmetric?** The range `[-5, 10]` suggests the expected value function output is skewed positive. Given the reward structure (tracking rewards ~0.75 each, penalties like `base_height_l2` at -10.0, etc.), it's not immediately obvious why the positive side needs twice the headroom. Could this be tuned more tightly, or does it need to accommodate accumulated discounted returns?

3. **Clamping `returns_batch` directly** modifies the regression targets, which introduces bias into the value function estimate. Wouldn't it be safer to only clamp `value_batch` (the network output) and leave the targets unclamped, so the value function can at least try to approximate extreme returns even if its output is bounded?

Any insight into how these bounds were chosen would be appreciated.
